index.html

```diff
@@ -3417,7 +3417,7 @@ <h2><span class="chunk">iCCP</span> Embedded ICC profile</h2>
           </table>
 
           <p>The profile name may be any convenient name for referring to the profile. It is case-sensitive. Profile names shall
-          contain only printable Latin-1 characters and spaces (only character codes 32-126 and 161-255 decimal are allowed).
+          contain only printable Latin-1 characters and spaces (only character codes 0x20-7E and 0xA1-FF are allowed).
```

Comment:
Do we want to say "character codes" or "byte values"? Unlike most legacy encodings, Latin-1 and ASCII map directly to the first 256 code points of Unicode, so it is convenient to talk in terms of Unicode's jargon here vs. having to talk about bytes => characters => code points. That is, you could say:

> ...spaces (only code points U+0020 - U+007E and U+00A1 - U+00FF are allowed)

Failing that (and being consistent with the current text) I'd probably write:

> ...spaces (that is, only byte values in the ranges 0x20-7E and 0xA1-FF are allowed).

Reply:
Changed to "code points"